test/utils.EventLogRiemannClient.spec.js

```diff
@@ -185,7 +185,54 @@ describe('utils', function () {
         expect(secondResponse).to.be.an('object');
         expect(secondResponse).to.eql(expectedSecondResultObject);
       });
+
+      it('should not log any event in case of bad request', function () {
+        const event = {
+          host: 'INLN50932351A',
```

Comment:
Can we add a dummy hostname, e.g. hostname, instead of the equipment number? :)

Reply:
Yes , ok :)